src/main/kotlin/tech/thatgravyboat/skyblockpv/api/data/SkyBlockProfile.kt

```diff
@@ -115,6 +117,41 @@ data class SkyBlockProfile(
                         skin = obj["skin"]?.takeIf { it !is JsonNull }?.asString,
                     )
                 },
+                trophyFish = run {
+                    val trophyFishData = member.getAsJsonObject("trophy_fish") ?: return@run TrophyFishData(mapOf(), null, 0, listOf())
+
+                    TrophyFishData(
```

Comment:
so this doesnt fill up too much can this be moved into a function in `TrophyFishData` called f`romJson`